code/modules/mob/living/say.dm

```diff
@@ -170,7 +170,11 @@ proc/get_radio_key_from_channel(var/channel)
 	if (speaking)
 		// This is broadcast to all mobs with the language,
 		// irrespective of distance or anything else.
-		if(speaking.flags & HIVEMIND)
+		if(speaking.flags & HIVEMIND && src.get_species() == "Vaurca")
+			speaking.broadcast(src,trim(message))
+			src << "\blue You consign your thoughts to the hive-net."

```

Comment:
In regular conversation, won't this get annoying as hell? To constantly have, "You consign your thoughts to the hive-net" sent every time you input a message?